### PR TITLE
task/product-copies-i18n Moving the product copies into localized json

### PR DESCRIPTION
**Pull request by fqlx**
task/product-copies-i18n Moving the product copies into localized json

### CODE REVIEW

File: shared/src/components/Product/DesktopProductCatalogueGalleryView.tsx [hunk 1]
```diff
@@ -157,17 +163,24 @@ const DesktopProductCatalogueGalleryView: React.FC<
       );
     }
 
+    const asset = getAssets(currentVault);
+    console.log("asdf", isPutVault(currentVault), currentVault);
```
Comment (leonkho) [inline shared/src/components/Product/DesktopProductCatalogueGalleryView.tsx:167]: Console.log

File: shared/src/i18n/en/shared.json [hunk 2]
```diff
@@ -1,7 +1,23 @@
 {
-  "title": "Welcome to react using react-i18next and typescript 4.1",
-  "description": {
-    "part1": "This is an example using multiple namespaces.",
-    "part2": "😉"
+  "ProductCopies": {
+    "calldescription": "Generates yield by running an automated {{asset}} covered call strategy.",
```
Comment (leonkho) [inline shared/src/i18n/en/shared.json:3]: Would it be better for us to camelcase these?

Comment (fqlx) [reply shared/src/i18n/en/shared.json:3]: Sure we can.  It doesn't matter - style preference.
Maybe we camelcase with first letter capitalized to be consistent for all.

Comment (leonkho) [reply shared/src/i18n/en/shared.json:3]: Yeap this works, will be consistent with everything if so

**Review by cjinghong — APPROVED** [review]

**Review by cjinghong — DISMISSED** [review]

**Review by cjinghong — APPROVED** [review]

**Pull request merged**